### DISCUSSION TITLE
Configuration to disable compression of release assets

### DISCUSSION THREAD

**Discussion by dmca-glasgow** [tauri-apps/tauri#10593] [Q&A] [answered]
Hi there,

Apologies if I've missed it, but I haven't found a configuration option to allow me to disable the compression of built executables when using GitHub Actions.

My release assets currently look like this:
![Screenshot 2024-08-13 at 11 15 34](https://github.com/user-attachments/assets/39a2263f-bd06-4c11-9da5-1c7d4adf54fc)

My app is for accessibility purposes and I'm trying to make the download-and-run process as simple and straightforward as possible, and so would like the executables to not need uncompressed if possible.

Thanks.

**Comment by FabianLars** [comment, marked as answer]
AppImage.tar.gz and setup.nsis.zip are only for the Updater feature, your users are not meant to download these.

app.tar.gz (macos) however is used by the updater and users directly. The issue is that `.app` is actually a folder and you can't upload folders to github releases so we need to archive it first.
The only alternative to that would be to enable the .dmg target and let users download that instead. If you double click the dmg you will then have to drag-drop your app in a small window to install it but this is standard on macos so shouldn't be too bad.
The .tar.gz is still required for the updater then though.

If you also want the app exe instead of the setup exe on windows then you're asking for https://github.com/tauri-apps/tauri-action/issues/302 which is not implemented yet so you'd have to upload the exe yourself. Note that using the app exe instead of the installer comes with a set of caveats (most are mentioned in the linked issue).

**Reply by dmca-glasgow** [reply]
Hi @FabianLars, thanks for the response (and all your awesome work around here).

I found out most of these things since I asked the question.

Here's the solution I ended up with: https://github.com/dmca-glasgow/isos/releases/tag/v0.0.34

Implemented here: https://github.com/dmca-glasgow/isos/blob/main/.github/actions/post-release/src/index.ts

**Reply by FabianLars** [reply]
Oh that's cool! I didn't know the display name could be different than the file name in the release.
